nixos/modules/services/desktops/pipewire.nix

```diff
@@ -5,8 +5,18 @@ with lib;
 
 let
   cfg = config.services.pipewire;
-  packages = with pkgs; [ pipewire ];
+  enable32BitAlsaPlugins = cfg.alsa.support32Bit
+                           && pkgs.stdenv.isx86_64
+                           && pkgs.pkgsi686Linux.pipewire != null;
 
+  jack-libs = pkgs.runCommand "jack-libs" {} ''
+    mkdir -p "$out/lib"
+    ln -s "${pkgs.pipewire.jack}/lib" "$out/lib/pipewire"
+  '';
+  pulse-libs = pkgs.runCommand "pulse-libs" {} ''
+    mkdir -p "$out/lib"
+    ln -s "${pkgs.pipewire.pulse}/lib" "$out/lib/pipewire"
+  '';
```

Comment:
There are two reasons I didn't do that. One is that I don't like the package output should depend on module internals. That seems like a potential backwards compatibility issue in the future. We probably want to change the module eventually to avoid using `LD_LIBRARY_PATH` once a way is setup (and the inevitable mass recompilation acceptable). The second is that currently the output can be used as an override on FHS and wrapper style packages, the exact ones that tend to cause problems with this setup. For example the following works:

  ```nix
  pkgs.tor-browser-bundle-bin.override { libpulseaudio = pkgs.pipewire.pulse; }
  ```

This is, of course, only valid when development information like headers aren't required.